example/speech-demo/decode_mxnet.sh

```diff
@@ -70,7 +70,7 @@ feats="scp:$sdata/JOB/feats.scp"
 inputfeats="$sdata/JOB/mxnetInput.scp"
 
 
-if [ ! -f $sdata/1/feats.scp ]; then
+if [ -f $sdata/1/feats.scp ]; then
```

Comment:
it depends on the purpose of this line 

#### avoid reconstucting

if want to avoid reconstructing,  this line must be

```
if [ ! -f $sdata/1/mxnetinput.scp ]; then
``` 

but the contents of mxnetinput.scp is just one line:

```
NO_FEATURE_TRANSFORM scp:/home/sooda/speech/kaldi/egs/timit/s5/data/dev/split8/2/feats.scp
```

I think there is no need to avoid reconstructing. 

#### confirm the orig feat exist


mxnetinput.scp is a wrapper for feats.scp, so to cofirm the feats.scp exist or not is necessary. 

```
if [ -f $sdata/1/feats.scp ]; then
```